docs/machine-learning/resources/transforms.md

```diff
@@ -1,168 +1,102 @@
 ---
-title: Machine learning data transforms - ML.NET
+title: Data transformations | ML.NET
 description: Explore the feature engineering components supported in ML.NET.
-author: JRAlexander
-ms.custom: seodec18
-ms.date: 01/14/2019
+author: natke
+ms.author: nakersha
+ms.date: 04/02/2019
 ---
 
-# Machine learning data transforms - ML.NET
+# Data transformations
 
-The following tables contain information about all of the data transforms supported in ML.NET.
+Data transformations are used to prepare data for model training. The transformations in this guide return classes that implement the [IEstimator](xref:Microsoft.ML.IEstimator`1) interface. Data transformations may also be chained together. When chained, the specific data transformation class determines the data type and format the transformer expects from the previous element in the chain, and also the data type and format the transformation produces.
```

Comment:
I would give an example after this, to make it easier to understand:

For example, if you want to normalize (project in the 0-1 interval) multiple columns of your dataset, rather than use several normalization transformations, you can initially concatenate the columns together, than apply a normalize transform.

Reply:
Good idea